client/src/components/UI/DateRangePicker.jsx

```diff
@@ -18,6 +18,7 @@ const DateRangePicker = ({ startDate, endDate, setStartDate, setEndDate }) => {
         startDate={startDate}
         endDate={endDate}
         customInput={<DatePickerCustomInput />}
+        dateFormat="yyyy/MM/dd"
```

Comment:
Can the format use dashes like yyyy-MM-dd so the punctuation matches the date columns in the grid?

Reply:
They can! Easy fix.